src/framework/components/session/middleware.clj

```diff
@@ -0,0 +1,30 @@
+(ns framework.components.session.middleware
```

Comment:
On wednesday we decided to go with interceptors for the app component's route/controller context.
Also, after app component is merged, please update the sessions example to reflect code moved to the framework.